### PR TITLE
mktemp: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/mktemp.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+- Create an empty temporary file with a given suffix and print the absolute path to file:
 
-`mktemp -d`
+`mktemp --suffix "{{.ext}}"`
 
-- Create a temporary file with a specified suffix:
+- Create a temporary directory and print the absolute path to it:
 
-`mktemp --suffix "{{.txt}}"`
+`mktemp --directory`
```
Comment (nicokosi) [inline pages/common/mktemp.md:16]: I suggest not using long options that are visibly not supported on `macOS` (see `*`):
```suggestion
`mktemp -d`
```

\* : 
```sh
$ echo $OSTYPE
darwin20.0
$ mktemp --directory
mktemp: illegal option -- -
usage: mktemp [-d] [-q] [-t prefix] [-u] template ...
       mktemp [-d] [-q] [-u] -t prefix
```

Comment (EmilySeville7cfg) [reply pages/common/mktemp.md:16]: Are you using GNU Coreutils?

Comment (nicokosi) [reply pages/common/mktemp.md:16]: > Are you using [GNU Coreutils mktemp](https://www.gnu.org/software/coreutils/manual/coreutils.html#mktemp-invocation)?

No, I am using BSD's `mktemp` (which is the default one on macOS, I guess).

> I think we should prefer by default some specific set of utilities GNU/BSD or anything else not to mix them in an unpredictable way.

Not sure to understand. :innocent:
But other maintainers will probably have an opinion about it. :thought_balloon:

Comment (sbrl) [reply pages/common/mktemp.md:16]: Indeed, we do prefer long arguments where possible. However, in this instance they are not portable - so we should use the short ones which do seem to be portable.

Comment (EmilySeville7cfg) [reply pages/common/mktemp.md]: What to prefer:

- `(non-portable long option: --directory)` in command description as now
- `mktemp -d|--directory` supposing that all portable options are going first

?

File: pages/common/mktemp.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 # mktemp
 
 > Create a temporary file or directory.
-> More information: <https://www.gnu.org/software/autogen/mktemp.html>.
+> More information: <https://manned.org/mktemp.1>.
```
Comment (navarroaxel) [inline pages/common/mktemp.md:4]: Use the author's web.
```suggestion
> More information: <https://www.gnu.org/software/autogen/mktemp.html>.
```

**Review by navarroaxel — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Comment by git-em** [comment]
Would it make sense to add a parenthesis saying that the `--suffix` option is "GNU-only"?

**Pull request merged**